src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -541,7 +541,15 @@ private function formatControllerLink($controller, string $anchorText, callable
             if (null === $controller) {
                 return $anchorText;
             } elseif (\is_array($controller)) {
-                $r = new \ReflectionMethod($controller[0], $controller[1]);
+                try {
+                    $r = new \ReflectionMethod($controller[0], $controller[1]);
+                } catch (\ReflectionException $e) {
+                    if ('kernel' !== $controller[0] || !$getContainer || !\is_string($kernelClass = $getContainer()->getDefinition('kernel')->getClass())) {
```

Comment:
This part is optional, I don't know if we can safely get the kernel class like I did.